modules/dynamic-plugins/ref-rh-compatible-plugins.adoc

```diff
@@ -19,10 +19,10 @@ The following Technology Preview plugins are not preinstalled and must be instal
 |`https://access.redhat.com/downloads/content/480/ver=2.4/rhel---9/2.4/x86_64/product-software[@ansible/plugin-scaffolder-backend-module-backstage-rhaap]` |1.0.0 |
 
 
-|Orchestrator Frontend|`https://www.npmjs.com/package/@janus-idp/backstage-plugin-orchestrator[@janus-idp/backstage-plugin-orchestrator]` |1.22.6| https://www.parodos.dev/1.2-rc/docs/[Learn more]
+|Orchestrator Frontend|`https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-orchestrator[@red-hat-developer-hub/backstage-plugin-orchestrator]` |1.22.6| https://www.parodos.dev/1.2-rc/docs/[Learn more]
 
 |Orchestrator Backend
-|`https://www.npmjs.com/package/@janus-idp/backstage-plugin-orchestrator-backend-dynamic[@janus-idp/backstage-plugin-orchestrator-backend-dynamic]` |1.22.9 |
+|`https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-orchestrator-backend-dynamic[@red-hat-developer-hub/backstage-plugin-orchestrator-backend-dynamic]` |1.22.9 |
```

Comment:
latest at https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-orchestrator-backend (the non `-dynamic` version) is 5.1.0. SHould we use that for 1.4?

```suggestion
|`https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-orchestrator-backend-dynamic[@red-hat-developer-hub/backstage-plugin-orchestrator-backend-dynamic]` |5.1.0 |
```

Reply:
There are no dynamic plugin releases of orchestrator npmjs.com npm registry.
For the orchestrator operator used by customers, we release to https://npm.registry.redhat.com/@redhat/
And also there we don't have a compatible version for 1.4 yet
And even when we do, the documentation here is more complex, it requires also setting the npm registry configmap.
For 1.5 - I understand there will be releases of images of all plugins in rhdh-plugins repo, and that will include the orchestrator

Reply:
so I guess we shuold just remove any mention of orchestrator plugins for now? Or link to your https://github.com/redhat-developer/rhdh-plugins/tree/main/workspaces/orchestrator repo with a caveat that they're not ready for Backstage 1.32 yet?